README.md

```diff
@@ -58,42 +47,57 @@ Active Record to grab the article with the `id` that is in the `params` and set
 it equal to `@article`. Then, it should render the `show.erb` view page. That
 view should use ERB to render the `@article`'s title and content.
 
-### Update
+### Create
 
-The Update CRUD action corresponds to the edit controller action and view.
+Now it's time to set up the ability to create an article.
 
-Create a controller action, `get '/articles/:id/edit'`, that renders the view,
-`edit.erb`. This view should contain a form to update a specific article. The 
-form should `POST` to a controller action, `patch '/articles/:id'`.
+First, create a route in your controller:  `get '/articles/new'`, that renders the
+`new.erb` view.  This view will be a blank form that should submit a `POST` request
+to `/articles`.  (Look up the method and action attributes for HTML forms if you
+aren't sure how to do this).
 
-You'll need to make sure the edit form includes:
+Now you will need to tell your controller what to do when your form sends that
+`POST` request, so create a route on your controller `post '/articles'` that creates a new
+article from the `params` from the form, then redirects to that new article's show page.
 
+
+### Update
+
+The Update CRUD action corresponds to the edit controller action and view.
+
+Create a controller action, `get '/articles/:id/edit'`, that renders the view
+`edit.erb`. This view should contain a form to update a specific article--similar to the form
+you made for a new article, but the fields should be pre-populated with the existing title and
+content of the article. Although we want to send a `PATCH` request to `/articles/:id`, we have to
+be a little sneaky to trick HTML into letting us do something besides a `GET` or a `POST`.  
+Define the controller action `patch '/articles/:id'`, but your form will be configured to send
```

Comment:
This is mildly confusing to read (sentence structure). How about:
Define the controller action `patch '/articles/:id'`. Your form will be configured to send a `POST` request in the `method` attribute. Inside the form itself, add a hidden field to specify a `PATCH` request like so:

Reply:
Yeah, I had trouble with this. Better now?